src/main/java/org/opentripplanner/street/model/edge/StreetEdge.java

```diff
@@ -23,12 +23,12 @@
 import org.opentripplanner.routing.linking.DisposableEdgeCollection;
 import org.opentripplanner.routing.linking.LinkingDirection;
 import org.opentripplanner.routing.util.ElevationUtils;
+import org.opentripplanner.service.vehiclerental.street.RentalRestrictionExtension;
```

Comment:
We should split this interface and its implementations. The interface belongs to `org.opentripplanner.street.model`, while the implementations belong to `org.opentripplanner.service.vehiclerental.street`